src/Internal/ClassHelper.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace mglaman\PHPStanDrupal\Internal;
+
+use PhpParser\Node\Stmt\Class_;
+
+/**
+ * @internal
+ */
+final class ClassHelper
+{
+    public static function isInDrupalNamespace(Class_ $class): bool
+    {
+        // @phpstan-ignore-next-line
+        if (!isset($class->namespacedName)) {
+            return false;
+        }
+
+        return 'Drupal' === (string) $class->namespacedName->slice(0, 1);
+    }
+
+    public static function isSharedNamespace(Class_ $class): bool
```

Comment:
Should we add a comment to explain what is considered to be a shared namespace? 🤔 

When I read the code, it seems clear to me but I don't know if it is clear for others

Reply:
I think the code explains it pretty well, the first two parts of the namespace must match